samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/BotController.cs

```diff
@@ -111,8 +116,9 @@ public class BotController : ControllerBase
         // Upload chat history into chat repository and embeddings into memory.
 
         // 1. Create a new chat and get the chat id.
-        newChat = new ChatSession(userId, chatTitle);
+        newChat = new ChatSession(chatTitle);
         await this._chatRepository.CreateAsync(newChat);
+        await this._chatParticipantRepository.CreateAsync(new ChatParticipant(userId, newChat.Id));
```

Comment:
out of curiosity, if this fails will the created chat be empty/dangling?

Reply:
Yes, you are right! Good catch! I will fix this